packages/safe-upgrade/files/usr/sbin/safe-upgrade

```diff
@@ -336,6 +336,11 @@ local function upgrade(args)
     local boot_script = string.format(boot_script_tpl, fw_mtd_str, partitions.other)
     set_uboot_env(string.format('boot_%d', partitions.other), boot_script)
     set_testing_partition(partitions.other)
+
```

Comment:
For a future improvement, the upgrade function relies on args, and which args it uses are not explicit. would be better to make them explicit.

Reply:
Make them explicit how? Document it? Do you have an idea? Thanks!

Reply:
Ok I understand now, thanks! I will make the arguments explicit in a future PR